schema/src/CreditClassInfo.yaml

```diff
@@ -0,0 +1,90 @@
+id: https://framework.regen.network/schema/CreditClassInfo
+name: CreditClassInfo
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+  qudt: http://qudt.org/schema/qudt/
+  unit: http://qudt.org/vocab/unit/
+  xsd: http://www.w3.org/2001/XMLSchema#
+imports:
+  - taxonomy
+  - SDG
+  - ProjectBenefit
+  - PrimaryImpact
+default_curi_maps:
+  - semweb_context
+default_prefix: rfs
+default_range: string
+
+classes:
+  CreditClassInfo:
+    class_uri: rfs:CreditClassInfo
+    abstract: true
+    title: Credit Class info
+    description: Base class with common fields used to define credit class info.
+    slots:
+      - name
+      - description
+      - url
+      - sourceRegistry
+      - approvedMethodologies
+      - creditingTerm
+      - permanencePeriod
```

Comment:
I think we remove `permanencePeriod` from the root schema as this is carbon specific and even then not all of them have permanence periods

Reply:
In general we shouldn't have schemas for specific credit classes. If we need more specific schemas for subsets of `CreditClassInfo`, I took an attempt at a refactor that creates specific ones for `Biodiviersty` and `Carbon` credit types:

https://github.com/regen-network/regen-data-standards/pull/28/commits/7ee3104c60c5fd27a15eb6e3b3283ad3cd5ff14d

In this, I included `permanencePeriod` in the Carbon one.  I've done this refactor more as a discussion point, rather than being really set on this approach, but curious to hear your feedback.